kpops/components/base_components/kubernetes_app.py

```diff
@@ -107,35 +108,56 @@ def helm_release_name(self) -> str:
         """The name for the Helm release. Can be overridden."""
         return self.name
 
-    @override
-    def _validate_custom(self, **kwargs) -> None:
-        super()._validate_custom(**kwargs)
-        self.validate_kubernetes_name(self.name)
+    @property
+    def helm_chart(self) -> str:
+        """Return component's helm chart
+
+        :return: Helm chart of this component
+        :rtype: str
+        """
```

Comment:
I'd say `"""Return component's helm chart"""` is enough here. It's probably me who overexplained it like this :D